pkg/tasks/alertmanager.go

```diff
@@ -157,23 +167,8 @@ func (t *AlertmanagerTask) create(ctx context.Context) error {
 	}
 
 	{
-		// Create trusted CA bundle ConfigMap.
```

Comment:
Make sure that you delete the `trusted-ca-bundle` resource which would be coming from upgrade path. I had followed an approach which you can refer[1], but it might change based on the reviewer's opinion :)

[1] https://github.com/openshift/cluster-monitoring-operator/blob/edda4a35ff5a5b66f846b7e68b198380f3f393a7/pkg/tasks/prometheus.go#L390-L402

Reply:
Thanks :) I forgot to remove this resource when upgrading.  Will correct it now.